src/Oscoin/Storage/Block/SQLite/Internal.hs

```diff
@@ -156,15 +181,16 @@ getGenesisBlock
        , StorableTx c tx
        , FromField (Hash c)
        )
-    => Connection
+    => MVar Connection
```

Comment:
Should this also run in the `Transaction` monad?

Reply:
oops, yes.

Reply:
d3378c5